neanderthal/src/main/kotlin/au/com/outware/neanderthal/presentation/view/activity/VariantListActivity.kt

```diff
@@ -85,6 +88,21 @@ class VariantListActivity : AppCompatActivity(), VariantListPresenter.ViewSurfac
         navigate(EditVariantActivity::class, EditVariantPresenter.ViewSurface.EXTRA_NAME to name)
     }
 
+    override fun goToMainApplication() {
+        val resolvedInfos = packageManager.queryIntentActivities(Intent(Intent.ACTION_MAIN), PackageManager.GET_RESOLVED_FILTER)
```

Comment:
For example, if an application has multiple `ACTION_MAIN` activities does this logic cause all of them to be launched?